scraper/app/routes.py

```diff
@@ -24,6 +24,29 @@ def test():
     return "done"
 
 
+@app.route('/getnews/daterange/thread', methods=['POST'])
+def get_date_ranges_thread():
+
+    content = request.get_json()
+
+    news_list = []
+
+    for timespan in content:
+        print(timespan)
+
+        from_ = timespan['from']
```

Comment:
No need to have underscores after variable since "from" doesn't exist in the function.

Reply:
from is used by python, import from

Reply:
You showed him! @roca5461